src/miral/window_specification.cpp

```diff
@@ -88,8 +90,10 @@ miral::WindowSpecification::Self::Self(mir::shell::SurfaceSpecification const& s
     parent(spec.parent),
     input_shape(spec.input_shape),
     input_mode(),
-    shell_chrome(spec.shell_chrome)
-    ,confine_pointer(spec.confine_pointer)
```

Comment:
Doesn't match layout of surrounding code

Reply:
How so?

Reply:
comma placement

Reply:
??

After my changes, all commas are placed at the end of lines (consistent with both the rest of the file, and the style guide).

Reply:
My bad